src/Search.js

```diff
@@ -192,12 +192,14 @@ function Search(props) {
             </div>
           </div>
           
-          <header className="App-header recom">
-            <Recommendation
-              onPressCreate={onPressCreate}
-              recommendations={recommendations}
-            />
-          </header>
+          <div className="recom-body">
+            <header className="App-header recom">
+              <Recommendation
+                onPressCreate={onPressCreate}
+                recommendations={recommendations}
+              />
+            </header>
+          </div>
```

Comment:
Double check in resolve conflicts, asking which one to keep. Someone should be able to merge this pr afterwards.

Reply:
Yeah, it's correct. It's just to separate the login header from the recommendation body so the background image is scooted down.